### PR TITLE
sqlpp11: add version 0.61

### PR DESCRIPTION
**Pull request by sonicxml**
Specify library name and version:  **sqlpp11/0.61**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — DISMISSED** [review]

File: recipes/sqlpp11/all/test_package/CMakeLists.txt [hunk 1]
```diff
@@ -7,5 +7,5 @@ conan_basic_setup(TARGETS)
 find_package(Sqlpp11 CONFIG REQUIRED)
 
 add_executable(example example.cpp)
-set_property(TARGET example PROPERTY CXX_STANDARD 11)
+set_property(TARGET example PROPERTY CXX_STANDARD 14)
```
Comment (uilianries) [inline recipes/sqlpp11/all/test_package/CMakeLists.txt:10]: Why do you need C++14 now? I didn't find any information on https://github.com/rbock/sqlpp11/releases/tag/0.61

Comment (sonicxml) [reply recipes/sqlpp11/all/test_package/CMakeLists.txt:10]: I was getting the following error: `error: constexpr function's return type 'void' is not a literal type`

https://github.com/conan-io/conan-center-index/pull/8783

Comment (sonicxml) [reply recipes/sqlpp11/all/test_package/CMakeLists.txt:10]: They added the `constexpr void` in 0.61: https://github.com/rbock/sqlpp11/compare/0.60...0.61#diff-5341f4f4a83948ea9eee5499ef9ad879c329456f997156233974608d1dd7bf6eR37

Comment (uilianries) [reply recipes/sqlpp11/all/test_package/CMakeLists.txt:10]: Excellent, in this case, you need to add a restriction in the recipe too. The test package package can be prepared now, but users won't. I'll open a PR to you asap.

Comment (uilianries) [reply recipes/sqlpp11/all/test_package/CMakeLists.txt:10]: @sonicxml consider https://github.com/sonicxml/conan-center-index/pull/1

Comment (sonicxml) [reply recipes/sqlpp11/all/test_package/CMakeLists.txt:10]: Merged, thanks!

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**